test/SymbolGraph/Symbols/DocComment/BatchMode.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -module-name BatchMode -emit-module-path %t/BatchMode.swiftmodule -emit-symbol-graph -emit-symbol-graph-dir %t -O -enable-batch-mode -incremental
+// RUN: %FileCheck %s --input-file %t/BatchMode.symbols.json
+
+/// This is some func.
+public func someFunc() {}
+
+// CHECK: range
```

Comment:
Is it worth checking the location here in case "range" is added as a key somewhere else?

Reply:
The problem is that the symbol graphs that come from the compiler are not formatted, so (as far as i know) there’s no easy way to check for a given line range without looking for the entire `{“start”:4,”end”:15}` sequence all at once. is there a way to pretty print JSON in a test, or should i just throw the whole sequence into a check condition?